airflow/www/security.py

```diff
@@ -171,12 +172,26 @@ def __init__(self, appbuilder):
             view.datamodel = CustomSQLAInterface(view.datamodel.obj)
         self.perms = None
 
+    def init_role(self, role_name, perms):
+        """
+        Initialize the role with the permissions and related view-menus.
+        :param role_name:
+        :param perms:
+        :return:
+        """
+        warnings.warn(
+            "`init_role` has been deprecated. Please use `sync_default_roles` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        self.sync_default_roles([{'role': role_name, 'perms': perms}])
```

Comment:
Oh wait. the "default" part of the name probably doesn't make sense when called with custom role info. Time to rename it?

Reply:
Yeah... `sync_roles` is the natural fit, but it's taken. I'll figure something out.